src/components/taskDetails/index.tsx

```diff
@@ -69,11 +67,14 @@ const TaskDetails: FC<Props> = ({ taskID }) => {
     const isAuthorized = useContext(isUserAuthorizedContext);
 
     const [isEditing, setIsEditing] = useState<boolean>(false);
-    const [taskTitle, setTaskTitle] = useState<string[]>([]);
-    const [id, setId] = useState<string[]>([]);
-    const [isFetched, setIsFetched] = useState<boolean>(false);
     const initialDataRef = useRef<Record<string, any> | undefined>({});
     const { data, isError, isLoading } = useGetTaskDetailsQuery(taskID);
+    const taskDependencyIds: string[] = data?.taskData?.dependsOn || [];
+    const {
+        data: dependencyData,
+        isLoading: Loading,
```

Comment:
please avoid using capital case for constants

Reply:
done